model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -819,17 +837,21 @@ async def create_deepspeed_bundle(
                 )
             ).model_bundle_id
 
-    async def create_vllm_bundle(
+    def _create_vllm_bundle_command(
         self,
-        user: User,
         model_name: str,
         framework_image_tag: str,
-        endpoint_unique_name: str,
         num_shards: int,
         quantize: Optional[Quantization],
         checkpoint_path: Optional[str],
         chat_template_override: Optional[str],
+        multinode: bool,
+        is_leader: bool,
```

Comment:
nit: but kinda prefer if we negate this and provide an `is_worker` instead. That fits my mental model a bit better since we don't really think of leaders in the non-multinode case. We could also default is_worker to False so we don't have to pass it at all for the non multinode case